### ISSUE TITLE
Problems upgrading to Aurelia Beta 1 

### ISSUE THREAD

**Issue by adriatic** [aurelia-ui-toolkits/aurelia-kendoui-plugin#71]
This is a summary issue which includes several changes done all at once.

#### 1 updating all plugin and sample application references to Aurelia:

1.1 [root level package.json](https://github.com/aurelia-ui-toolkits/aurelia-kendoui-plugin/blob/master/package.json)
1.2 [sample level package.json](https://github.com/aurelia-ui-toolkits/aurelia-kendoui-plugin/blob/master/sample/package.json)
1.3 [sample level index.html](https://github.com/aurelia-ui-toolkits/aurelia-kendoui-plugin/blob/master/sample/index.html)

#### 2 changing the access to KendoUI library
The current code uses **bower endpoint** : https://github.com/kendo-labs/bower-kendo-ui, while until recently we used **github endpoint**: https://github.com/telerik/kendo-ui-core

The use of bower endpoint requires the use of [jspm-git](https://www.npmjs.com/package/jspm-git) module which caused some problems to @adriatic see 
- https://github.com/aurelia-ui-toolkits/aurelia-kendoui-plugin/issues/70
- https://github.com/aurelia-ui-toolkits/aurelia-kendoui-plugin/issues/65

If everyone agrees that using bower endpoint ensures better access to Kendo - we can do that now as I believe that the problems with jspm-git were all pilot errors. However I would recommend that we take a lot more careful approach to making changes to the code in a "one issue at the time" fashion.

**Comment by charlespockert** [comment]
These are both github endpoints and can be installed using jspm as standard

The private telerik repo requires the use of jspm-git 

The reason for needing the kendo labs repo is that it matches the format/layout of the pro repo when downloaded from the telerik private servers

If we mixed the telerik/kendo-ui-core with the private telerik repo the code wouldn't work as the files would not be in their expected places in the file system

**Comment by adriatic** [comment]
I understand all these issues now and have no problems going either way. At the same time, we need to make sure that developers using our plugin are served best by including two likely scenarios:

1. Plugin user already has Kendo PRO - in this case, he would probably prefer to plop Kendo bits into a `vendor` folder and our instructions should point to the place where a single line needs to be uncommented in order to enable plugin's access to kendo bits
2. Plugin user does not have Kendo PRO - in this case using the private telerik repo (where the user needs to present the password) **might** make sense if Telerik issues such password. My recollection is that Telerik issues a licence number which might not be used to authenticate the access to that telerik repo. 

In summary, I would like to verify the case 2 (without me having to buy KendoUI PRO) - but rather by talking to Telerik sales,

**Comment by adriatic** [comment]
**Solution proposal**: Let's go the way Jeroen suggested earlier today in Gitter (https://gitter.im/adriatic/Aurelia-UI-Toolkits?at=564dd1bf7ae518a71d2fd935). Anyone against it please add to this thread with explanation why would that not be good.

**Comment by charlespockert** [comment]
Can anyone please comment on how they think dropping kendo into a vendor folder is going to work?

Our import statements refer to single kendo components, wouldn't this mean that all imports for kendo files would need to be directed to the kendo.all.min bundled js?

Does kendo.all.min export the kendo components in a way that system js can import

**Comment by charlespockert** [comment]
To clarify, can this:

`import 'kendo-ui/src/js/kendo.button';`

Be pointed at kendo.all.min js in the vendor folder?

I assume a mapping is required, wouldn't this be a manual process?

**Comment by JeroenVinke** [comment]
@adriatic can you close this since we have https://github.com/aurelia-ui-toolkits/aurelia-kendoui-plugin/issues/75 to address the concerns of @charlespockert and we have https://github.com/aurelia-ui-toolkits/aurelia-kendoui-plugin/issues/77 to start the implementation of jspm-git from scratch?

**Comment by adriatic** [comment]
Agreed and am closing this
